### PR TITLE
Add example lookup table and data file

### PR DESCRIPTION
**Pull request by dashaaa**
### Background

Lookup tables and their content can be uploaded using panther_analysis_tool, we need examples to be public.

### Changes

Added example lookup table and it's content

### Testing

panther_analysis_tool test-lookup-table --path example_cidr_lookup_table.yml

### CODE REVIEW

File: templates/example_cidr_lookup_table.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+AnalysisType: lookup_table # always lookup_table
+LookupName: simple_cidr_lookup # str
+Enabled: true # bool
+Description: Lookup table description # str (Optional)
+# FileName: ./relative/path/to/content.csv # str (Optional)
```
Comment (rleighton) [inline templates/example_cidr_lookup_table.yml:5]: Why is this commented?

Comment (dashaaa) [reply templates/example_cidr_lookup_table.yml:5]: Oops, uncommenting, sorry about that

**Comment by imju-panther** [comment]
@lindsey-w can you approve this? Thx!

File: templates/example_cidr_lookup_content.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+network,geoname_id
+1.0.0.0/24,2077422
+1.0.1.0/24,1814991
+1.0.2.0/23,1814991
+1.0.4.0/22,2077456
+1.0.8.0/21,1814991
+1.0.16.0/20,1814991
```
Comment (k-bailey) [inline templates/example_cidr_lookup_content.csv:7]: nit: newline

**Review by k-bailey — APPROVED** [review]

File: templates/example_cidr_lookup_content.csv (hunk 2 above)
Comment (dashaaa) [reply templates/example_cidr_lookup_content.csv]: Done!

Comment (dashaaa) [reply templates/example_cidr_lookup_content.csv]: Sorry, don't know how to submit after commit without re-review

**Review by lindsey-w — APPROVED** [review]

**Pull request merged**